AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.html

```diff
@@ -39,9 +39,10 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
       <div
         [ngClass]="updateJudgeAttempted && (!isJudgeSelected || isJudgeParticipantError) ? 'govuk-form-group--error' : 'govuk-form-group'"
       >
-        <label class="govuk-label" for="participantEmail" id="judgeEmail-error"> Judge or courtroom account </label>
+        <label class="govuk-label" for="{{ judgeLocator }}" id="judgeEmail-error"> Judge or courtroom account </label>
         <app-search-email
           [disabled]="false"
+          locator="{{ judgeLocator }}"
```

Comment:
I would change this as well to data binding same as the other change.

Reply:
will do